src/openrct2/drawing/TTFSDLPort.cpp

```diff
@@ -357,13 +357,13 @@ static TTF_Font* TTF_OpenFontIndexRW(FILE* src, int freesrc, int ptsize, long in
         {
             fclose(src);
         }
-        return NULL;
+        return nullptr;
```

Comment:
I deliberately left out this file because it's directly ~~stolen~~taken from SDL2-TTF.

Reply:
Then wouldn't this fit better with the other dependencies, or inside the thirdparty folder?